unicodetools/src/main/java/org/unicode/xml/XMLProperties.java

```diff
@@ -0,0 +1,482 @@
+package org.unicode.xml;
+
+import com.ibm.icu.dev.util.UnicodeMap;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.*;
+import java.util.Map.Entry;
```

Comment:
Please don't import nested classes; call sites are easier to understand if they say `Map.Entry` rather than `Entry`.

Reply:
I think that I fixed all of these.